src/test/java/io/stargate/sgv2/jsonapi/api/v1/tables/DropIndexIntegrationTest.java

```diff
@@ -0,0 +1,80 @@
+package io.stargate.sgv2.jsonapi.api.v1.tables;
+
+import static io.stargate.sgv2.jsonapi.api.v1.util.DataApiCommandSenders.assertNamespaceCommand;
+import static io.stargate.sgv2.jsonapi.api.v1.util.DataApiCommandSenders.assertTableCommand;
+
+import io.quarkus.test.common.WithTestResource;
+import io.quarkus.test.junit.QuarkusIntegrationTest;
+import io.stargate.sgv2.jsonapi.testresource.DseTestResource;
+import org.junit.jupiter.api.*;
+
+@QuarkusIntegrationTest
+@WithTestResource(value = DseTestResource.class, restrictToAnnotatedClass = false)
+@TestClassOrder(ClassOrderer.OrderAnnotation.class)
+class DropIndexIntegrationTest extends AbstractTableIntegrationTestBase {
+
+  String simpleTableName = "simple_table_drop_index_test";
+
+  @BeforeAll
+  public final void createSimpleTable() {
+    String tableJson =
+            """
+                              {
+                                      "name": "%s",
+                                      "definition": {
+                                          "columns": {
+                                              "id": {
+                                                  "type": "text"
+                                              },
+                                              "age": {
+                                                  "type": "int"
+                                              },
+                                              "name": {
+                                                  "type": "text"
+                                              }
+                                          },
+                                          "primaryKey": "id"
+                                      }
+                            }
+                        """
+            .formatted(simpleTableName);
+
+    assertNamespaceCommand(keyspaceName).postCreateTable(tableJson).wasSuccessful();
```

Comment:
I think Aaron changed things so that `createTable()` would be same as "postCreateTable()" and ".wasSuccessful()", and if so could simplify this
(basic idea being that "createXxx()" are for expected success cases and "postCreateXxx()" only need to be used directly when testing fail conditions).